### DISCUSSION TITLE
linking to DockerHub account

### DISCUSSION THREAD

**Discussion by kjyoder** [l3atbc-datadog/pushkin#83] [General] [answered]
I've followed almost all of the Pushkin setup steps at https://languagelearninglab.gitbook.io/pushkin/getting-started/deploying-to-aws

Near the end, it says to run
$ pushkin setDockerHub

However, when I run this command, I get the following error:
error: unknown command 'setDockerHub'. See 'pushkin --help'.

Is there a different way to link to DockerHub?

**Comment by conbainbridge** [comment, marked as answer]
Hello,

Apologies for the confusion - this particular section we intend to fix to be up-to-date and will cover during our Thursday workshop session - this command hasn't gone live just yet, but will be up ASAP!

**Reply by jkhartshorne** [reply]
@kjyoder yes, sorry about that. It's in the updated CLI, which works locally on my computer but hasn't been pushed to production. We'll comment it out in the documentation.

**Comment by restmegan** [comment]
Hi @kjyoder ! I just wanted to check in to see how Pushkin has been going for you. The Pushkin team is online today answering questions so if there's anything you need help with please let us know!
